packages/shared/src/utils/getOpenseaExternalUrl.ts

```diff
@@ -12,11 +12,12 @@ export const hexHandler = (str: string) => {
   return d;
 };
 
-export const getOpenseaExternalUrl = (contractAddress: string, tokenId: string) => {
+export const getOpenseaExternalUrl = (contractAddress: string, tokenId: string, chainStr: string = "ethereum") => {
```

Comment:
looks like typescript linter is complaining about a small thing: when you default to a string literal, the type can be inferred, and therefore you could just do `contractAddress: string, tokenId: string, chainStr = 'ethereum'`, without specifying the type for the final param

<img width="1064" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/814986c0-6a6a-418b-93c4-a61b381e2eea">

that said, i suggest _not_ adding a default parameter and keeping it `chainStr: string`, and forcing the function caller to pass in a chain explicitly

also this is purely aesthetics, but i think the order of parameters should be:

`chainStr: string, contractAddress: string, tokenId: string`

Reply:
oh and to check locally if you have any typescript issues, you can run:
`moon run web:typecheck`